app/AppKernel.php

```diff
@@ -70,6 +70,10 @@ public function registerBundles()
             new AppBundle\AppBundle(),
         ];
 
+        // Symfony 3.3 moved server:* commands to another bundle
```

Comment:
That won't work  i think. For Symfony < 3.3 the functions (`server:run`, ...) works on demand. From Symfony 3.3 those stuff comes from `WebServerBundle`, which does not exist befor. So what won't work, or i don't know how, is to require conditional dependencies. 
This way it works as Symfony requires it by its own.